### ISSUE TITLE
aggregate_time is bugged 

### ISSUE THREAD

**Issue by danklotz** [danklotz/visCOS#5]
Aggregate time does currently only work if every aggregation unit has data. 

We build a work-around for Johannes, but that one only works for his special case (and destroys many other cases).

**Comment by chrisschuerz** [comment]
Is it the really the case? I think it depends on the aggregation function you provide. 
Maybe give a minimum example that does not work. 
It is a good point anyways we should consider for generalization

**Comment by danklotz** [comment]
Yep. I mean "bug" is probably the wrong word and it depends on what you expect from the function... The question is, what do when one of your aggregation units does not have any data.
